### PR TITLE
Us116068/handling permission errors

### PR DESCRIPTION
**Pull request by TomMichielsD2L**
image

### CODE REVIEW

**Comment by louiscoste** [comment]
If the user does not have permission then we should not be telling the user to try again later (text below box)

File: src/components/d2l-teacher-course-creation-admin.js [hunk 1]
```diff
@@ -274,7 +273,7 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
 			return html`
 				<d2l-tcc-error
 				.errorMessage="${this.localize('adminPermissionsError')}"
-				.renderBack="${false}">
+				renderBack>
```
Comment (louiscoste) [inline src/components/d2l-teacher-course-creation-admin.js:276]: Did you mean to have `hideBack`?
```suggestion
				hideBack>
```

Comment (TomMichielsD2L) [reply src/components/d2l-teacher-course-creation-admin.js]: yeah... fixed

**Comment by TomMichielsD2L** [comment]
> If the user does not have permission then we should not be telling the user to try again later (text below box)

This is to account for the admin being in the middle of the setup process.

File: src/components/d2l-teacher-course-creation-admin.js [hunk 2]
```diff
@@ -261,7 +269,13 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
 			</div>
 		`;
 
-		if (isEmpty) {
+		if (this.permissionError) {
+			return html`
+				<d2l-tcc-error
+				error-message="${this.localize('adminPermissionsError')}"
```
Comment (josephsaffron) [inline src/components/d2l-teacher-course-creation-admin.js:275]:
```suggestion
					error-message="${this.localize('adminPermissionsError')}"
```

File: src/components/d2l-teacher-course-creation-admin.js [hunk 3]
```diff
@@ -261,7 +269,13 @@ class TeacherCourseCreationAdmin extends BaseMixin(LitElement) {
...
+				hide-back>
```
Comment (josephsaffron) [inline src/components/d2l-teacher-course-creation-admin.js:276]:
```suggestion
					error-message="${this.localize('adminPermissionsError')}"
					hide-back>
```

Comment (josephsaffron) [reply src/components/d2l-teacher-course-creation-admin.js:276]: nit

**Pull request merged**